packages/typespec-ts/test/integration/generated/mediaTypes/src/parameters.ts

```diff
@@ -93,7 +94,8 @@ export interface GetByOverloadParentBodyParam {
     | string
     | Uint8Array
     | ReadableStream<Uint8Array>
-    | NodeJS.ReadableStream;
+    | NodeJS.ReadableStream
+    | string;
```

Comment:
This is introduced by getSchemaFromType util and issue tracked here https://github.com/Azure/autorest.typescript/issues/2273.